auth-lib/src/main/java/com/spotify/sdk/android/auth/app/SpotifyNativeAuthUtil.java

```diff
@@ -30,46 +30,30 @@
 import android.content.pm.PackageManager;
 import android.content.pm.Signature;
 
+import com.spotify.sdk.android.auth.AuthorizationRequest;
+import com.spotify.sdk.android.auth.LoginActivity;
+
 import java.io.UnsupportedEncodingException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
-public class SpotifyNativeAuthUtil {
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_VERSION;
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_CLIENT_ID;
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_REDIRECT_URI;
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_REQUESTED_SCOPES;
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_RESPONSE_TYPE;
+import static com.spotify.sdk.android.auth.IntentExtras.KEY_STATE;
 
-    /*
-     * This is used to pass information about the protocol version
-     * to the AuthorizationActivity.
-     * DO NOT CHANGE THIS.
-     */
-    private static final String EXTRA_VERSION = "VERSION";
+public class SpotifyNativeAuthUtil {
 
     /*
      * Constants below have their counterparts in Spotify app where
-     * they're used to parse messages received from SDK. 
+     * they're used to parse messages received from SDK.
      * If anything changes in the structure of those messages this version
-     * should be bumped and new protocol version should be implemented on the other side. 
+     * should be bumped and new protocol version should be implemented on the other side.
```

Comment:
this comment should be moved with the constants, right?

Reply:
The first part yes, I will clean up or rephrase